### DISCUSSION TITLE
2022/10/analyzing-va-townhall-comments/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#770] [Specific Post Comments]
# 2022/10/analyzing-va-townhall-comments/

Analyzing Reactions, and providing a little of my own.

https://seankilleen.com/2022/10/analyzing-va-townhall-comments/

**Comment by SeanKilleen** [comment]
I've had one comment from someone offline asking for an analysis of how many full comments are within 98% of each other. 

> "Almost all of these get astro turfed the moment they're open, which is the point of them."

**Comment by SeanKilleen** [comment]
FYI, I've updated the post to add a link to the data file -- and [you can find it here.](https://1drv.ms/u/s!AuaqsrDoMOUHofNXPofXp4xRlm1yYg?e=QaFIYf) I'll try to keep it fresh.

**Comment by SeanKilleen** [comment]
I've updated the data this evening; available at the same link.

**Reply by SeanKilleen** [reply]
Comments by Date

| Date | Total | `Support` in Subject | `Oppose` in Subject|
| ---- | ----- | ------------------ | -----------------|
|2022-09-26| 14882| 1140 |7387|
|2022-09-27| 8969| 3014 |3310|
|2022-09-28| 26674| 5722 |5782|
|2022-09-29| 1959| 554 |809|
|2022-09-30| 1144| 332 |440|
|2022-10-01| 629| 206 |176|
|2022-10-02| 483| 124 |165|
|2022-10-03| 795| 261 |268|
|2022-10-04| 509| 159 |152|
|2022-10-05| 573| 191 |145|
|2022-10-06| 596| 126 |88|
|2022-10-07| 313| 87 |60|
|2022-10-08| 186| 47 |38|
|2022-10-09| 163| 50 |39|
|2022-10-10| 508| 116 |109|

ℹ️ NOTE: This is rudimentary. For example, "support" in subject could mean "support trans youth" or "support this policy". take with a grain of salt.

**Comment by SeanKilleen** [comment]
Looks like they've changed the format of the town hall site in a way that breaks the scraping in this blog. I'll try to update soon.

_____

Update: clicking the "show all comments on one page" link still works. Going to use that instead. Edited the directions.

**Comment by SeanKilleen** [comment]
## Total count of comments by Day

| Date | # Comments |
| ---- | ---------  |
| 2022-09-26 | 14882 |
| 2022-09-27 | 8969 |
| 2022-09-28 | 26674 |
| 2022-09-29 | 1959 |
| 2022-09-30 | 1144 |
| 2022-10-01 | 629 |
| 2022-10-02 | 483 |
| 2022-10-03 | 795 |
| 2022-10-04 | 509 |
| 2022-10-05 | 573 |
| 2022-10-06 | 600 |
| 2022-10-07 | 313 |
| 2022-10-08 | 186 |
| 2022-10-09 | 163 |
| 2022-10-10 | 518 |
| 2022-10-11 | 269 |
| 2022-10-12 | 244 |
| 2022-10-13 | 276 |
| 2022-10-14 | 265 |
| 2022-10-15 | 101 |
| 2022-10-16 | 116 |
| 2022-10-17 | 166 |
| 2022-10-18 | 201 |
| 2022-10-19 | 299 |
| 2022-10-20 | 568 |
| 2022-10-21 | 1159 |
| 2022-10-22 | 432 |
| 2022-10-23 | 550 |
| 2022-10-24 | 1379 |
| 2022-10-25 | 1708 |
| 2022-10-26 | 5126 |
| 2022-10-27 | 13 |
